tracer/test/test-applications/integrations/Samples.XUnitTests/Samples.XUnitTests.csproj

```
@@ -4,10 +4,13 @@
     <IsPackable>false</IsPackable>
     <OutputType>Library</OutputType>
     <ApiVersion Condition="'$(ApiVersion)' == ''">2.5.0</ApiVersion>
+    <MicrosoftTestSDKVersion Condition="$(ApiVersion.StartsWith('2.6'))">16.7.1</MicrosoftTestSDKVersion>
```

Coulnd't that work? And hopefully limit next changes (I know from the previous versions I'm pessimistic).

```suggestion
    <MicrosoftTestSDKVersion Condition="'$(ApiVersion)'&gt;='2.4'">16.7.1</MicrosoftTestSDKVersion>
```

---

Yeah, probably 🤷 I _would_ fix it if doing so wouldn't require another round of CI and approvals 🙈